example/lib/indicator.dart

```diff
@@ -0,0 +1,44 @@
+import 'dart:math';
+
+import 'package:streamdeck/streamdeck.dart';
+
+/// A simple action that shows an indicator.
+class IndicatorAction extends StreamDeckPluginAction {
+  static const actionUuid = 'com.dantup.streamdeck.example.indicator';
+
+  IndicatorAction(super.plugin, super.context);
+
+  int _value = 42;
+
+  void sendValue() => setFeedback({
+    'value': _value.toString(),
+    'indicator': _value.toString(),
+  });
```

Comment:
Is this correct?

Reply:
Oh, I see what you mean now. `SetFeedbackPayload` is just a type alias:

```dart
typedef SetFeedbackPayload = Map<String, String>;
```

Unlike all of the others, this payload is just a map of key/values so there's not much value in its own class. I created the `typedef` so that we can use `SetFeedbackPayload` in the signatures (which may be useful if more of this is code-gen'd in future), but since it's just a map, using a map literal here is correct.

That said, there are some built-in (non-custom) layouts that we could perhaps create classes for (which would give some extra safety/typing/completion if you're not using custom layouts), but we'd need to add them in a way that still supports custom layouts. We could add a `// TODO` about that for now to avoid holding up landing this basic support and update this later.

Reply:
Ah I see, thanks!

Yes for the built-in layouts that would indeed be great!